executor/write_test.go

```diff
@@ -261,8 +261,8 @@ func (s *testSuite) TestInsert(c *C) {
 	// issue 6424
 	tk.MustExec("drop table if exists t")
 	tk.MustExec("create table t(a time(6))")
-	tk.MustExec("insert into t value('20070219173709.055870'), ('20070219173709.055'), ('-20070219173709.055870'), ('20070219173709.055870123')")
```

Comment:
Why change this case?

Reply:
If we try to insert a DateTime literal, the input should not carry a sign. This behaviour is `MySQL` compatible.